sonic-psud/scripts/psud

```diff
@@ -0,0 +1,188 @@
+def get_platform_and_hwsku():
+    try:
+        proc = subprocess.Popen([SONIC_CFGGEN_PATH, '-H', '-v', PLATFORM_KEY],
+                                stdout=subprocess.PIPE,
+                                shell=False,
+                                stderr=subprocess.STDOUT)
+        stdout = proc.communicate()[0]
+        proc.wait()
+        platform = stdout.rstrip('\n')
+
+        proc = subprocess.Popen([SONIC_CFGGEN_PATH, '-d', '-v', HWSKU_KEY],
+                                stdout=subprocess.PIPE,
+                                shell=False,
+                                stderr=subprocess.STDOUT)
+        stdout = proc.communicate()[0]
+        proc.wait()
+        hwsku = stdout.rstrip('\n')
+    except OSError, e:
+        raise OSError("Cannot detect platform")
+
+    return (platform, hwsku)
+
+
+# Loads platform specific psuutil module from source
+def load_platform_psuutil():
+    global platform_psuutil
+
+    # Get platform and hwsku
+    (platform, hwsku) = get_platform_and_hwsku()
+
+    # Load platform module from source
+    platform_path = PLATFORM_ROOT_DOCKER
+    hwsku_path = "/".join([platform_path, hwsku])
+
+    try:
+        module_file = "/".join([platform_path, "plugins", PLATFORM_SPECIFIC_MODULE_NAME + ".py"])
+        module = imp.load_source(PLATFORM_SPECIFIC_MODULE_NAME, module_file)
+    except IOError, e:
+        log_error("Failed to load platform module '%s': %s" % (PLATFORM_SPECIFIC_MODULE_NAME, str(e)), True)
+        return -1
+
+    try:
+        platform_psuutil_class = getattr(module, PLATFORM_SPECIFIC_CLASS_NAME)
+        platform_psuutil = platform_psuutil_class()
+    except AttributeError, e:
+        log_error("Failed to instantiate '%s' class: %s" % (PLATFORM_SPECIFIC_CLASS_NAME, str(e)), True)
+        return -2
+
+    return 0
+
+def psu_db_update(psu_tbl, psu_num):
+    for psu_index in range(1, psu_num + 1):
+        fvs = swsscommon.FieldValuePairs([('presence',
+                                           'true' if platform_psuutil.get_psu_presence(psu_index) else 'false'),
+                                          ('status',
+                                           'true' if platform_psuutil.get_psu_status(psu_index) else 'false')])
+        psu_tbl.set("PSU{}".format(psu_index), fvs)
+
+#=============================== Main ================================
+
+def main():
+    log_info("Starting up...")
+
+    # Register our signal handlers
+    signal.signal(signal.SIGHUP, signal_handler)
+    signal.signal(signal.SIGINT, signal_handler)
+    signal.signal(signal.SIGTERM, signal_handler)
+
+    # Load platform-specific psuutil class
+    err = load_platform_psuutil()
+    if err != 0:
+        log_error("failed to load psuutil")
+        sys.exit(1)
+
+    state_db = swsscommon.DBConnector(swsscommon.STATE_DB,
+                                     REDIS_HOSTNAME,
+                                     REDIS_PORT,
+                                     REDIS_TIMEOUT_MSECS)
```

Comment:
Add one more leading space to lines 165-167 so the arguments visually align with the first argument on line 164.